install_config/topics/vsphere_prerequisites.adoc

```diff
@@ -3,128 +3,136 @@
-template inherit this property.
-====
+==== Dynamic Provisioning Permissions
 
-. Create and assign roles to the vSphere Cloud Provider user and vSphere
-entities. vSphere Cloud Provider requires the following privileges to interact
-with vCenter.
-+
-[cols="4*", width="100%",options="header"]
+*Note:* dynamic persistent volume creation is the recommended practice.
+
+[cols=4*,width="100%",options=header]
 |===
 |Roles
 |Privileges
 |Entities
-|Propagate to Children
+|Propergate to Children
 
 |manage-k8s-node-vms
-|Resource.AssignVMToPool
-System.Anonymous
-System.Read
-System.View
-VirtualMachine.Config.AddExistingDisk
-VirtualMachine.Config.AddNewDisk
-VirtualMachine.Config.AddRemoveDevice
-VirtualMachine.Config.RemoveDisk
-VirtualMachine.Inventory.Create
-VirtualMachine.Inventory.Delete
-|Cluster,
-Hosts,
-VM Folder
+|Resource.AssignVMToPool, VirtualMachine.Config.AddExistingDisk, VirtualMachine.Config.AddNewDisk, VirtualMachine.Config.AddRemoveDevice, VirtualMachine.Config.RemoveDisk, VirtualMachine.Inventory.Create, VirtualMachine.Inventory.Delete, VirtualMachine.Config.Settings
+|Cluster, Hosts, VM Folder
 |Yes
 
 |manage-k8s-volumes
-|Datastore.AllocateSpace
-Datastore.FileManagement
-System.Anonymous
-System.Read
-System.View
+|Datastore.AllocateSpace, Datastore.FileManagement (Low level file operations)
 |Datastore
 |No
 
 |k8s-system-read-and-spbm-profile-view
-|StorageProfile.View
-System.Anonymous
-System.Read
-System.View
+|StorageProfile.View (Profile-driven storage view)
 |vCenter
 |No
 
-|ReadOnly
-|System.Anonymous
-System.Read
-System.View
-|Datacenter,
-Datastore Cluster,
-Datastore Storage Folder
+|Read-only (pre-existing default role)
+|System.Anonymous, System.Read, System.View
+|Datacenter, Datastore Cluster, Datastore Storage Folder
 |No
 
 |===
 
-See the
-link:https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html[vSphere
-Documentation Center] for steps to create a custom role, user, and role
-assignment.
+==== Static Provisioning Permissions
+
+*Note:* Datastore.FileManagement is only required for the role manage-k8s-volumes, if PVC is created to bind with statically provisioned PV, and reclaim policy set to delete. When PVC is deleted, associated statically provisioned PV will also be deleted.
+
+[cols=4*,width="100%",options=header]
+|===
+|Roles
+|Privileges
+|Entities
+|Propergate to Children
+
+|manage-k8s-node-vms
+|VirtualMachine.Config.AddExistingDisk, VirtualMachine.Config.AddNewDisk, VirtualMachine.Config.AddRemoveDevice, VirtualMachine.Config.RemoveDisk
+|VM Folder
+|Yes
+
+|manage-k8s-volumes
+|Datastore.FileManagement (Low level file operations)
+|Datastore
+|No
+
+|Read-only (pre-existing default role)
+|System.Anonymous, System.Read, System.View
+|vCenter, Datacenter, Datastore Cluster, Datastore Storage Folder, Cluster, Hosts
+|No
+...
+|===
+
+
+
+=== Procedure
```

Comment:
If the procedure's starting here, you need to break some of the preceding information into another module. Procedures can only have ~3 sentences of information before the prereqs and the procedure steps. All other detailed conceptual and reference material needs to go in other modules.

Reply:
Prior to this is the permission table required to support the procedure, is that okay?